src/components/postView/view/postDisplayView.tsx

```
@@ -216,8 +257,26 @@ const PostDisplayView = ({
+    [
+      stickyWrapperStyle,
+      post,
+      parentType,
+      handleUpvotePress,
+      handlePayoutDetailsPress,
+      handleVotersIconPress,
+      activeVotesCount,
+      cacheVoteIcrement,
+      _handleOnReblogsPress,
+      isLoggedIn,
+      _showQuickReplyModal,
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)